contracts/series/SeriesController.sol

```diff
@@ -1203,4 +1219,127 @@ contract SeriesController is
             emit AllowedExpirationUpdated(timestamps[i]);
         }
     }
+
+    /// @notice This function allows the owner address to update allowed strikes for the auto series creation feature
+    /// @param strikeUnderlyingToken underlying asset token that options are written against
+    /// @param min minimum strike allowed
+    /// @param max maximum strike allowed
+    /// @param increment price increment allowed - e.g. if increment is 10, then 100 would be valid and 101 would not be (strike % increment == 0)
+    /// @dev Only the owner address should be allowed to call this
+    function updateAllowedTokenStrikeRanges(
```

Comment:
Yeah, I think Andre has an idea to improve this in the future based off volatility params and automate it instead of setting it explicitly.  For now we said we would just have the "owner" manually set the params for each asset.

Reply:
I think we want the range to be percentage of the underlying price. Otherwise we'd have to update it every time the price moves.

Reply:
Also, I think this logic should live in `SeriesDeployer`. SeriesController doesn't set any limits on what series can be created. It is the SeriesDeployer that is responsible for maintaining the rules.